tests/core/test_actions.py

```diff
@@ -75,22 +82,10 @@ def template_nlg():
 
 
 @pytest.fixture(scope="module")
-def template_sender_tracker(default_domain):
+def template_sender_tracker(default_domain: Domain):
     return DialogueStateTracker("template-sender", default_domain.slots)
 
 
-def test_text_format():
```

Comment:
why did we remove this test?

Reply:
@wochinge could you please take this over

Reply:
it tested the `__str__` method of events. I don't think it makes sense to test that `str(object)` works as expected in Python. Another reason for this test could be to pin the formatting of these events to a specific value. However, I checked and it's not used for anything critical in our codebase so pinning this isn't necessary.